src/constants_and_utils/messageBuilder.js

```diff
@@ -64,6 +64,12 @@ function getDefaultTextTextNotPresentOnElementMessage(htmlElement, text) {
   }'.`;
 }
 
+function getParameterIsMissingForFunctionMessage(param, fn) {
```

Comment:
With the suggested adjustment I think it becomes clearer what each parameter means.
What do you think?
```suggestion
function getParameterIsMissingForFunctionMessage(missedParameter, missedFunction) {
```

Reply:
I'd prefer to go with the smaller parameter names. Also, there is no need to say `missed` in the parameter names since the function name already mentions that it gets the parameter that is **missing** for a given function